schema-engine/sql-schema-describer/src/lib.rs

```diff
@@ -825,8 +825,8 @@ pub enum DefaultKind {
 }
 
 impl DefaultValue {
-    pub fn db_generated(val: impl Into<String>) -> Self {
-        Self::new(DefaultKind::DbGenerated(Some(val.into())))
+    pub fn db_generated<S: Into<String>>(val: impl Into<Option<S>>) -> Self {
```

Comment:
There's already an `Into` in the generic parameter `<S: Into<String>>`, so I wonder why we need `val: impl Into<Option<S>>` as well? Wouldn't it be sufficient to have `val: Option<S>` or something like it?

Reply:
The issue here is the difference in what we provide from the `unsupported` path vs for regular `PrismaValue`s. Having both `Into`s allowed us to change the function to accept what we provide from both paths without needing to change the args we pass everywhere it's used in our tests

Reply:
Thanks for the additional context! Feel free to merge